gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -562,7 +593,41 @@ export class AdvancedDynamicTexture extends DynamicTexture {
         this.onEndLayoutObservable.clear();
         super.dispose();
     }
+
+    /** Creates the GUI editor window. */
+    private _createGUIEditor() {
+        this.BJSGUIEDITOR.GUIEditor.Show({
+            liveGuiTexture: this
+        });
+    }
+
+    /**
+     * Launch the node material editor
+     * @param config Define the configuration of the editor
+     * @return a promise fulfilled when the node editor is visible
+     */
+     public edit(config?: IGUIEditorOptions): Promise<void> {
```

Comment:
could we also allow passing in the options the GUIEditor type ? so that in case of es6 ppl would still have a way to use this function ?